modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogZoneDescriptor.java

```diff
@@ -151,6 +160,20 @@ public int replicas() {
         return replicas;
     }
 
+    /**
+     * Return quorum size.
+     */
```

Comment:
Let's add more context somewhere, what quorum size is and how it is related to consensus group size (i.e. voters count) and total replicas count.